lib/extensions/require_nested.rb

```diff
@@ -15,6 +15,13 @@ def require_nested(name)
       Object.require_nested name
     end
   end
+
+  def require_nested_all
+    Dir.glob("#{Dir.pwd}/app/models/#{self.name.underscore}/*.rb").each do |full_path|
```

Comment:
`.sort.each`.  We want a consistent order to requires since `Dir.glob` order is system dependent.

Also, some of the originals were not being required in alphabetical order so this would change that.  This is probably good but could cause a problem if they were out of order for a reason.  We should fix that reason and keep it requiring in order.

Reply:
I believe `Dir.pwd` is wrong.  We should use `Rails.root.join` or if we want to be support non-rails, it should use `Pathname.new(__dir__).join...`

Reply:
Got it.
And now the caller passes in `__FILE__` because this could be called from within a gem.